tet.rs/src/tetris/board/piece/mod.rs

```diff
@@ -0,0 +1,7 @@
+pub mod helper;
+pub mod kick_tables;
+pub mod piece_matrix;
+pub mod piece_position;
+pub mod piece_representation;
+pub mod piece_rotation;
+pub mod piece_type;
```

Comment:
You could probably keep most mods private, and only pub use what you need, otherwise the module separation doesn't make much sense.

Reply:
I guess what you're saying is exact.

Later on, I plan to have a pub struct in `tetris/` that's used to actually play the game. This would reference (future) helpers such as a `PieceMove` that would handle the movement of a piece, rather than trying to move around with integers that frankly don't make much sense.

For the time being, I'm keeping these modules public so that I can quickly test stuff from `main`, but they will later be made private.